src/controllers/ReviewPenaltyController.ts

```diff
@@ -0,0 +1,104 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { Penalty, PenaltyList } from 'ch-sdk-node/dist/services/lfp';
+import { controller } from 'inversify-express-utils';
+import { SafeNavigationBaseController } from './SafeNavigationBaseController';
+
+import { AuthMiddleware } from 'app/middleware/AuthMiddleware';
+import { Appeal } from 'app/models/Appeal';
+import { OTHER_REASON_DISCLAIMER_PAGE_URI, PENALTY_DETAILS_PAGE_URI, REVIEW_PENALTY_PAGE_URI } from 'app/utils/Paths';
+
+const template = 'review-penalty';
+const navigation = {
+    previous(): string {
+        return PENALTY_DETAILS_PAGE_URI;
+    },
+    next(): string {
+        return OTHER_REASON_DISCLAIMER_PAGE_URI;
+    }
+};
+
+type TableColumn = {
+    text: string;
+};
+type TableRow = TableColumn[];
+type Table = {
+    caption: string,
+    header: TableRow;
+    madeUpToDate: string;
+    tableRows: TableRow[];
+};
+
+@controller(REVIEW_PENALTY_PAGE_URI, SessionMiddleware, AuthMiddleware)
+export class ReviewPenaltyController extends SafeNavigationBaseController<Table> {
+
+    public static PENALTY_EXPECTED_ERROR: string = 'Penalty object expected but none found';
+
+    constructor() {
+        super(template, navigation);
+    }
+
+    public prepareViewModelFromAppeal(appeal: Appeal): Record<string, any> & Table {
+
+        if (!appeal.penaltyIdentifier.penaltyList) {
+            throw new Error(ReviewPenaltyController.PENALTY_EXPECTED_ERROR);
+
+        }
+        return { ...appeal.penaltyIdentifier, ...this.createTable(appeal.penaltyIdentifier.penaltyList) };
+    }
+
+    private createTable(penalties: PenaltyList): Table {
```

Comment:
Similar to the suggestion above, createTable isnt a great name imo. Even if we go for penaltyListTable?